mne/io/nicolet/tests/test_nicolet.py

```diff
@@ -0,0 +1,42 @@
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import inspect
+import numpy as np
+from numpy.testing import assert_array_almost_equal, assert_array_equal
+
+from mne import concatenate_raws
+from mne.io import read_raw_nicolet, Raw
+from mne.utils import _TempDir
+
+FILE = inspect.getfile(inspect.currentframe())
+base_dir = op.join(op.dirname(op.abspath(FILE)), 'data')
+fname = op.join(base_dir, 'test_nicolet_raw.data')
+
+
+def test_data():
+    """Test reading raw nicolet files."""
+    tempdir = _TempDir()
+    read_raw_nicolet(fname, preload=False)
+
+    # Make sure concatenation works
+    raw = read_raw_nicolet(fname, preload=True)
```

Comment:
should probably loop with `preload=True` and `preload=False` to make sure your non-preload mode works